iOS/MMIWG2S/ViewControllers/IntroViewController.swift

```diff
@@ -0,0 +1,206 @@
+        labelMMIW.centerXAnchor.constraint(equalTo:self.view.centerXAnchor),
+        labelMMIWInfo.heightAnchor.constraint(equalToConstant: 150),
+        labelMMIWInfo.leadingAnchor.constraint(equalTo: self.view.leadingAnchor, constant:25),
+        labelMMIWInfo.trailingAnchor.constraint(equalTo: self.view.trailingAnchor, constant:-25),
+        labelMMIWInfo.bottomAnchor.constraint(equalTo: spacerView1.topAnchor, constant:0),
+        labelMMIWInfo.centerXAnchor.constraint(equalTo:self.view.centerXAnchor),
+        spacerView1.leadingAnchor.constraint(equalTo: self.view.leadingAnchor, constant:25),
+        spacerView1.trailingAnchor.constraint(equalTo: self.view.trailingAnchor, constant:-25),
+        spacerView1.heightAnchor.constraint(equalToConstant:0),
+        spacerView1.bottomAnchor.constraint(equalTo:bottomView.topAnchor, constant:0),
+        bottomView.widthAnchor.constraint(equalToConstant: 100),
+        bottomView.heightAnchor.constraint(equalToConstant: 55),
+        bottomView.trailingAnchor.constraint(equalTo: self.view.trailingAnchor, constant: -10),
+        bottomView.bottomAnchor.constraint(equalTo: spacerView2.topAnchor, constant:0),
+        spacerView2.leadingAnchor.constraint(equalTo: self.view.leadingAnchor, constant:25),
+        spacerView2.trailingAnchor.constraint(equalTo: self.view.trailingAnchor, constant:-25),
+
+        spacerView2.heightAnchor.constraint(equalToConstant:0),
+        spacerView2.bottomAnchor.constraint(equalTo:self.view.bottomAnchor, constant:0),
+        
+        // page control and button constraints
+        pageControlViews[0].heightAnchor.constraint(equalToConstant: 15),
+        pageControlViews[0].centerYAnchor.constraint(equalTo:bottomView.centerYAnchor),
+        pageControlViews[0].trailingAnchor.constraint(equalTo: bottomView.trailingAnchor, constant: -61),
+        pageControlViews[0].widthAnchor.constraint(equalToConstant: 2),
+
+        pageControlViews[1].heightAnchor.constraint(equalToConstant: 15),
+        pageControlViews[1].centerYAnchor.constraint(equalTo:bottomView.centerYAnchor),
+        pageControlViews[1].trailingAnchor.constraint(equalTo: bottomView.trailingAnchor, constant: -57),
+        pageControlViews[1].widthAnchor.constraint(equalToConstant: 2),
+
+        pageControlViews[2].heightAnchor.constraint(equalToConstant: 15),
+        pageControlViews[2].trailingAnchor.constraint(equalTo: bottomView.trailingAnchor, constant: -53),
+        pageControlViews[2].widthAnchor.constraint(equalToConstant: 2),
+        pageControlViews[2].centerYAnchor.constraint(equalTo:bottomView.centerYAnchor),
+
+        pageControlViews[3].heightAnchor.constraint(equalToConstant: 15),
+        pageControlViews[3].centerYAnchor.constraint(equalTo:bottomView.centerYAnchor),
+        pageControlViews[3].trailingAnchor.constraint(equalTo: bottomView.trailingAnchor, constant: -49),
+        pageControlViews[3].widthAnchor.constraint(equalToConstant: 2),
```

Comment:
sooo I think we could remove the index hardcoding here. We can remove these from the array and simplify this whole thing to
```
let leftMostTrailingPadding: CGFloat = 61
for (i, pageControlView) in pageControlViews.enumerated() {
    pageControlView.heightAnchor.constraint(equalToConstant: 15).isActive = true
    pageControlView.centerYAnchor.constraint(equalTo:bottomView.centerYAnchor).isActive = true
    let trailingPadding = CGFloat(i) * 4 - leftMostTrailingPadding
    pageControlView.trailingAnchor.constraint(equalTo: bottomView.trailingAnchor, constant: trailingPadding).isActive = true
        pageControlView.widthAnchor.constraint(equalToConstant: 2).isActive = true
}
```

Which gives us a _lot_ more flexibility if pages are added or removed.